src/modules/company/repository/company-repository.ts

```diff
@@ -36,7 +36,9 @@ export class CompanyRepository extends Repository<CompaniesEntity> {
     return this.findOne(id).catch(handleError);
   }
 
-  async UpdateCompanyById(company: CompaniesEntity, data: UpdateCompanyDto) {
+  async UpdateCompanyById(id: string, data: UpdateCompanyDto) {
+    const company = await this.findOne(id).catch(handleError);
+
     return this.save({
```

Comment:
usa o this.update

Reply:
volt ei apenas para como estava antes. Não alterei.